src/lib/kvbCheckVc.ts

```diff
@@ -80,6 +81,7 @@ export function buildKbvCheckArtifacts(
       GOV_UK_VC_DESCRIPTION,
       "Knowledge Based Verification (KBV) Identity Check"
     )
+    .addDatetime(GOV_UK_VC_CREATED_AT, new Date())
```

Comment:
Similarly here - there's `DCTERMS.created` which we've used in [the access logs demo](https://github.com/alphagov/di-solid-prototype/blob/main/src/controllers/access.ts#L54)